src/components/Identity/createComponent.js

```diff
@@ -104,7 +105,7 @@ export default (
             payload.expectResponse();
             if (
               config.thirdPartyCookiesEnabled &&
-              isThirdPartyCookieSupported(window)
+              isThirdPartyCookieSupported(getBrowser(window))
```

Comment:
Rather than `isThirdPartyCookieSupported` using `getBrowser` under the hood, I decided to pass in the browser. It makes it easier to unit test.